ui/component/claimListDiscover/view.jsx

```diff
@@ -155,6 +158,10 @@ function ClaimListDiscover(props: Props) {
     }
   }
 
+  if (!showReposts) {
+    options.claim_type = 'stream';
```

Comment:
Great. The SDK docs seem to suggest you can only supply a string.

Will update later / tomorrow morning UK time

Reply:
For some reason the app won't start for me to test these changes. I'll push tomorrow :+1:

Reply:
@seanyesmunt this seems to work for claim list, but not claim search. Looking through the SDK PR's looks like https://github.com/lbryio/lbry-sdk/pull/2777 is the change for list, but nothing for search.

Reply:
I added a comment here earlier then deleted it because I thought I was wrong... But I don't think I was - the SDK seems to only support this for the claim_list endpoint: https://github.com/lbryio/lbry-sdk/pull/2777
So I've left this as is.

Reply:
I gotcha. I left a comment on that asking for the other endpoints to be added too

Reply:
@Lukewh After talking with @tzarebczan, I think this is supported, but you need to be on the latest version of the SDK. If you are using lbry.tv, it might not be

Reply:
I'm just downloading the latest release of the sdk from the github releases page - I guess I should pull and build myself

Reply:
That works, also this project downloads the sdk specified in the package json when you run `yarn`. I just updated it to 60 so it should work if you rebase

Reply:
late to the party, but a better design for this would probably be:

if (defined) { 
  diff array
} else {
  make default
}

Reply:
I originally had all the code be the negative form of "hideReposts" which would allow what you suggest, but the rest of the naming takes the positive form (from what I can tell) of "showThing" so I conformed :)